core/event.mli

```diff
@@ -44,9 +44,14 @@ module Ok : sig
           ; dst : Location.t
           } (** Represents an event collected from Intel PT. *)
       | Power of { freq : int } (** Power event collected by Intel PT. *)
-      | Sample of
+      | Stacktrace_sample of
           { callstack : Location.t list (** Oldest to most recent calls in order. *) }
-          (** Represents event collected through sampling. *)
+          (** Represents cycles event collected through sampling. *)
+      | Event_sample of
+          { location : Location.t
+          ; period : int
```

Comment:
I don't think `period` is quite the right name in this context. `count` is a bit more accurate, though it is very much a function of the selected period.

Reply:
I think perf does actually call this field period though. I can change it to count though as it feels more accurate.